.gitignore

```diff
@@ -19,3 +19,4 @@ pkg/
 /tmp/
 /yarn-error.log
 /test-reports/
+.DS_Store
```

Comment:
:scissors:

Reply:
Done. Is there a reason to not gitignore this?

Reply:
@ankurp this has been proposed a few times before, reasoning here https://github.com/rails/rails/pull/22696